rollup.go

```diff
@@ -16,12 +16,12 @@ import (
 
 // Renames a file with the current timestamp
 func renameFileTimestamp(cfg *Config) (string, error) {
-	t := time.Now()
+	newFileName := time.Now().UTC().Format("2006-01-02T15-04-05.00000") + ".log"
```

Comment:
Just to designate the time part as per the examples at https://tools.ietf.org/html/rfc3339#section-5.8

I put the hyphens into the time rather than colons because it's more compatible with Windows and in places I've worked, it's quite common to copy log files from Linux to Windows.